### ISSUE TITLE
Generate QR to export vaccination certificates to another device

### ISSUE THREAD

**Issue by shiraneyo** [corona-warn-app/cwa-wishlist#582]
## Feature description
The Corona-Warn-App should offer the option to generate a QR code to export its stored vaccination certificates to other mobile devices.

## Problem and motivation
Currently the only way to export the vaccination certificate to another device is to scan the QR code of the "EU Digital Covid Vaccination Certificate" generated by the doctor or vaccination center. However, if such document is lost or not printed out for the patient(e.g. if the patient was simply told at the doctor's to scan the QR on their display), the Corona-Warn-App offers no way to export the certificates it has stored, making the user unable to switch mobile devices.

**Comment by Ein-Tim** [comment]
@shiraneyo 

You can just scan the QR-Code shown in the Corona-Warn-App with the CovPass app or any other app that's able to manage digital COVID certificates.

**Comment by shiraneyo** [comment]
@Ein-Tim, you're right, I misunderstood the CWA documentation(specifically [this CWA FAQ answer](https://www.coronawarn.app/en/faq/?search=lose#vac_cert_multiple_scan_possible) to mean that you would need to scan the original vaccination certificate(the PDF generated at the doctor's or vaccination center) to load it into a new device. I just tested this with a CWA installation on a second smartphone, and scanning the CWA vaccination QR code does allow importing it.

In my defense, first I tried to scan a screenshot of the CWA QR code with the CovPass-App on the same phone, and that did not work at all, the CovPass-App did not recognize the QR code as a valid vaccination certificate to import. So I assumed the CWA vaccination QR code was actually not meant for export into other devices.

Anyway, I'm closing this issue since the feature is already there. Thanks a lot for the quick reply! :)
